### PR TITLE
fix(): skip recaptcha checking bot traffic

### PR DESCRIPTION
**Pull request by tx0c**
it has triggered many good gmail registration requests flagged as bot, need logs to know why

### CODE REVIEW

File: src/connectors/gcp/index.ts [hunk 1]
```diff
@@ -101,11 +101,11 @@ class GCP {
 
     // fail for less than 0.5
     if (score < 0.5) {
-      return false
+      console.log("very likely bot traffic:", data)
```
Comment (guoliu) [inline src/connectors/gcp/index.ts:104]: Shall we use `[logger](https://github.com/thematters/matters-server/blob/develop/src/common/logger.ts#L58)` here? If sentry is still accessable.

**Review by guoliu — APPROVED** [review]

File: src/connectors/gcp/index.ts (hunk 1 above)
Comment (tx0c) [reply src/connectors/gcp/index.ts:104]: I'm aware of that, but:
1. sentry is logging only warn, error, above severe logs
2. the logger.info can only log a string, not any extra parameters, or will have to use some thing like `JSON.stringify(data)` to include data; for simple logging, it's always easier to call console.log; let AWS infra configure something to monitor console logs to automatically save to CloudWatch might be a a better way

**Comment by tx0c** [comment]
#2468

File: src/connectors/gcp/index.ts (hunk 1 above)
Comment (guoliu) [reply src/connectors/gcp/index.ts:104]: Sounds good, let's merge it in!

**Pull request merged**